### PR TITLE
Chores

### PR DESCRIPTION
**Pull request by iBug**
- Run `goimports` that `gofmt`s and sorts imports.
- Remove privilege-dropping code as PIDFile is not required for `Type=simple` services.

The first commit only contains `goimports` changes and can be safely ignored. Actual changes are carried out in subsequent commits.

### CODE REVIEW

**Comment by ZenithalHourlyRate** [comment]
@iBug Please also add RuntimeDirectory so that I can deploy it

**Comment by iBug** [comment]
RuntimeDirectory is not required as we no longer have a `PIDFile=` setting.

**Comment by ZenithalHourlyRate** [comment]
@iBug I need some pid file since I need to udpate it whenever json update

https://github.com/mirrorz-org/mirrorz-d-extension/blob/master/update.sh

**Comment by iBug** [comment]
Since you'd have to update that script as well, why not change it to:

```shell
systemctl show --property MainPID --value mirrorzd.service
```

I.E. you're asking systemd to give you the PID of the running service.

Or better yet, just use `systemctl kill -s SIGHUP something.service` to deliver the signal.

File: 302.go [hunk 1]
```diff
@@ -942,7 +905,6 @@ func main() {
 				LoadConfig(*configPtr, *debugPtr)
 			case syscall.SIGUSR2:
 				logger.Infof("Got A USR2 Signal! Now Reopen log file....\n")
-				InitLoggers()
```
Comment (ZenithalHourlyRate) [inline 302.go]: This is for logrotate...

Comment (iBug) [reply 302.go]: This one is duplicated. `InitLoggers()` is called twice consecutively and the first error is even dropped. I only removed the first call.

**Pull request merged**